test_integration/erc20_gateway/00_deployment/00_deployment.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const assert = require('assert');
+import BN from 'bn.js';
+
+import shared from '../shared'
+import Interacts from '../../../interacts/Interacts';
+
+describe('Contract deployment', async (): Promise<void> => {
+
+  it('deploy value token', async (): Promise<void> => {
+    shared.totalTokenSupply = new BN('8000000000000000000000');
+    shared.accounts = await shared.web3.eth.getAccounts();
+    shared.depositor = shared.accounts[2];
+    shared.facilitator = shared.accounts[3];
+    const valueToken = await shared.artifacts.ERC20Token.new(
+      shared.depositor,
+      shared.totalTokenSupply.toString(10),
+    );
+    shared.contracts.ValueToken.instance = Interacts.getERC20I(shared.web3, valueToken.address);
+    shared.contracts.ValueToken.address = valueToken.address;
+
+    const inContractInitialTokenBalance = await shared.contracts.ValueToken.instance.
+    methods.balanceOf(
+        shared.depositor,
+      )
+    .call();
+
+    assert.strictEqual(
+      shared.totalTokenSupply.eq(new BN(inContractInitialTokenBalance)),
+      true,
+      `Total token supply must be ${shared.totalTokenSupply.toString(10)} `
+      + `but got ${inContractInitialTokenBalance}`,
+    );
+  });
+
+  it('deploy origin anchor', async (): Promise<void> => {
+    const originAnchor = await shared.artifacts.Anchor.new();
+    assert.strictEqual(
```

Comment:
This assertion is not adding much value.

Reply:
Done